pages/accreditation/[slug].vue

```diff
@@ -1,21 +1,50 @@
 <template>
   <div class="page">
-    <section class="bg-gradient-to-b from-sushi-50 to-sushi-100 md:pt-28 max-w-screen ">
-      <div class="page-fade-in contain flex flex-col justify-center items-center max-w-8xl pt-28 pb-20 md:pb-0 lg:pt-12 gap-10">
-        <h1>{{ accred?.data.hero_title || "Your money is building a greener future" }}</h1>
+    <section class="bg-gradient-to-b from-sushi-50 to-sushi-100 pt-28 max-w-screen">
+      <div class="page-fade-in relative contain pb-20 lg:pt-12 z-10">
+        <article class="relative flex flex-col items-center border rounded-2xl w-full max-w-7xl min-h-[504px] px-4 py-10 bg-white text-center gap-10">
```

Comment:
I agree here that the article is not the best for this case but a wrapper is needed because the `contain` class uses padding instead of margin. The white box (article tag) needs to use padding. div parent should be using margin instead. See the video for what I mean:



Uploading Screen Recording 2024-09-11 at 11.06.10.mov…